packages/fx-core/resource/package.nls.json

```diff
@@ -387,15 +387,18 @@
   "core.common.OutlookWebClientName1": "Outlook web access client id 1",
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
   "core.common.CancelledMessage": "Operation is cancelled.",
-  "core.common.SpecVersionNotSupported": "OpenAPI version lower than 3.0.0 is not supported.",
-  "core.common.NoServerInformation": "No server information is found in the OpenAPI specification file.",
-  "core.common.MultipleServerInformation": "Multiple server information is found in the OpenAPI specification file.",
+  "core.common.NoServerInformation":  "No server information is found in the OpenAPI specification file.",
   "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
   "core.common.MissingOperationId": "Missing operationIds: %s.",
-  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file.",
+  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file: only GET and POST methods are supported and there can be at most one required parameter and no auth is allowed.",
   "core.common.AdditionalPropertiesNotSupported": "'additionalProperties' is not supported, and will be ignored.",
   "core.common.SchemaNotSupported": "'oneOf', 'anyOf', and 'not' schema are not supported: %s.",
   "core.common.UnknownSchema": "Unknown schema: %s.",
+  "core.common.UrlProtocolNotSupported": "Server url is not correct: protocol %s is not supported, you should use https protocol instead.",
+  "core.common.RelativeServerUrlNotSupported": "Server url is not correct: relative server url is not supported.",
```

Comment:
consider remove "Server" if it is not necessary.

Reply:
Same above